lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -251,6 +256,134 @@ div.course-wrapper {
         }
       }
     }
+    .proctored-exam-skip-confirm-wrapper {
+      padding: $baseline 30px;
+      background-color: rgb(242, 244, 245);
+      border-left: 5px solid $red;
+      .msg-title {
+        margin-bottom: 12px;
+        font-size: 22px;
+        font-weight: 600;
+      }
+      .msg-content {
+        color: rgb(121, 118, 118);
+        strong {
+          font-weight: 600;
+        }
+      }
+      .proctored-exam-skip-actions {
+        margin-top: $baseline;
+        @include float(right);
+      }
+
+      .btn {
+        transition: color .125s ease-in-out 0s,border-color
+        .125s ease-in-out 0s,background
+        .125s ease-in-out 0s,box-shadow
+        .125s ease-in-out 0s;;
+
+        // Display: inline, side-by-side
+        display: inline-block;
+        border-style: solid;
+        border-radius: 3px;
+        border-width: 1px;
+        font-weight: 600;
+
+        // Display: block, one button per line, full width
+        &.block {
+            display: block;
+            width: 100%;
+        }
+
+        // STATE: is disabled
+        &:disabled,
+        &.is-disabled {
+          pointer-events: none;
+          outline: none;
+          cursor: default;
+        }
+      }
+      .btn-base {
+        padding: 10px $baseline;
+        font-size: 16px;
```

Comment:
Please use [our variable](https://github.com/edx/edx-platform/blob/master/lms/static/sass/elements/_typography.scss#L92) for this.